dbos/system_database.go

```diff
@@ -1115,16 +1122,16 @@ func (s *systemDatabase) notificationListenerLoop(ctx context.Context) {
 
 const _DBOS_NULL_TOPIC = "__null__topic__"
 
-type workflowSendInputInternal struct {
-	destinationID string
-	message       any
-	topic         string
+type WorkflowSendInputInternal struct {
```

Comment:
Terrible name that must be made public for the `DBOSContext` to be implementable (`DBOSContext.Send` must take a typeless input)